bind/motor-mobile/bind.go

```diff
@@ -1,28 +1,25 @@
 package motor
 
 import (
-	"errors"
 	"fmt"
 
 	mtr "github.com/sonr-io/sonr/pkg/motor"
 	"github.com/sonr-io/sonr/pkg/motor/x/object"
-	mt "github.com/sonr-io/sonr/third_party/types/motor"
-	"github.com/sonr-io/sonr/x/registry/types"
+	ct "github.com/sonr-io/sonr/third_party/types/common"
+	mt "github.com/sonr-io/sonr/third_party/types/motor/api/v1"
+	rt "github.com/sonr-io/sonr/x/registry/types"
 	_ "golang.org/x/mobile/bind"
 )
 
 var (
-	errWalletNotExists = errors.New("mpc wallet does not exist")
-)
-
-var (
-	instance mtr.MotorNode
-	callback MotorCallback
+	objectBuilders map[string]*object.ObjectBuilder
+	instance       mtr.MotorNode
+	callback       MotorCallback
 )
 
 type MotorCallback interface {
 	OnDiscover(data []byte)
-	OnWalletCreated(ok bool)
+	OnWalletEvent(msg string, isDone bool)
```

Comment:
I think having a method for each event makes sense. It's more verbose and readable.

Reply:
+ 1

Reply:
refactored to use constants